definitions/bulk-action/json/AuthorisationCaseField.json

```diff
@@ -61,5 +61,12 @@
     "CaseFieldID": "PronouncementJudge",
     "UserRole": "caseworker-divorce-courtadmin_beta",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE_BulkAction",
+    "CaseFieldID": "D8DocumentsGenerated",
```

Comment:
is minipetition called d8?

Reply:
Not sure what you mean, but my other comment might make the reason I chose this name clearer.